### ISSUE TITLE
hacky solution to css var with unit

### ISSUE THREAD

**Issue by xiaofan2406** [tkh44/emotion#171]
Some context:
I am trying to covert a project using `react-jss` to use `emotion`. All the variables for height, fontsize, margin and etc are in JS integer.

- `emotion` version: 6.03
- `react` version: 15.6.1

Relevant code.

```javascript
const height = 100;
const className = css`
    height: ${height}px;
`;
```
This doesn't work as expect. I'm not sure if I missed anything.

And I had to do this for all the variables:
```javascript
const height = 100;
const className = css`
    height: calc(${height} * 1px);
`;
```
I am curious if there is any other solution to this problem

**Comment by tkh44** [comment]
I've been seeing weird behavior from the parser as well. I think I have a reproducible case in the `glamor-extended` branch (the reason the tests are failing right now).

**Comment by tkh44** [comment]
This has been fixed with #154
